cloudygames/serializers.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers
+from cloudygames.models import Game, PlayerSaveData, GameSession, UserGames
+
+class GameSerializer(serializers.HyperlinkedModelSerializer):
+	class Meta:
+		model = Game
+		fields = ('name', 'publisher', 'max_limit', 'address', 'user')
+
+	def create(self, validated_data):
```

Comment:
why all of the serializer here overwrite the `create` and `update`? I believe this is the default value, so you may not need any overwrite. Unless you have any specific things you overwrite.